src/esp/assets/ResourceManager.cpp

```diff
@@ -275,9 +276,17 @@ bool ResourceManager::loadScene(
       }
 
       // GLB Mesh
-      else if (info.type == AssetType::MP3D_MESH) {
+      else if (info.type == AssetType::MP3D_MESH ||
+               info.type == AssetType::UNKNOWN) {
         GltfMeshData* gltfMeshData =
             dynamic_cast<GltfMeshData*>(meshes_[mesh_i].get());
+        if (gltfMeshData == nullptr) {
+          Corrade::Utility::Debug()
+              << "AssetInfo::AssetType type error: unsupported physical type, "
+                 "aborting. Try running without \"--enable-phyisics\" and "
```

Comment:
```suggestion
                 "aborting. Try running without \"--enable-physics\" and "
```

Reply:
haha, thanks for catching this. Fixed on master.